salt/dex/ensure-dex-running.sls

```diff
@@ -1,16 +1,13 @@
 
 ensure_dex_running:
   # Wait until the Dex API is actually up and running
-  cmd.run:
-    - name: |
-        {% set dex_api_server = pillar['api']['server']['external_fqdn'] -%}
-        {% set dex_api_port = pillar['dex']['node_port'] -%}
-        {% set dex_api_server_url = 'https://' + dex_api_server + ':' + dex_api_port -%}
+  http.wait_for_successful_query:
+    {% set dex_api_server = pillar['api']['server']['external_fqdn'] -%}
+    {% set dex_api_port = pillar['dex']['node_port'] -%}
+    - name:       {{ 'https://' + dex_api_server + ':' + dex_api_port }}/.well-known/openid-configuration
+    - wait_for:   300
+    #- cert:       [{{ pillar['ssl']['crt_file'] }}, {{ pillar['ssl']['key_file'] }}]
```

Comment:
Quite sure we do not need these certs...

Reply:
I'd be in favour of removing it instead of commenting it out.